core/tests/block.rs

```diff
@@ -269,7 +269,7 @@ fn empty_block_serialized_size() {
 	let b = new_block(vec![], &keychain, &builder, &prev, &key_id);
 	let mut vec = Vec::new();
 	ser::serialize_default(&mut vec, &b).expect("serialization failed");
-	let target_len = 1_107;
```

Comment:
Are these different because of the updated `AUTOMATED_TESTING` constants above?

Reply:
Yes, this method has a limitation - a compressed POW must be at least 64 bits long because we read into u64, our test parameters were too low.